packages/flutter_markdown/lib/src/_functions_web.dart

```diff
@@ -84,3 +80,9 @@ Widget _handleDataSchemeUri(
   }
   return const SizedBox();
 }
+
+final String _userAgent = globalContext
```

Comment:
I'm guessing you can't use `package:web`?

Reply:
was trying to avoid it since it's for ONE SINGLE property access.

Reply:
Another alternative is using a dotted top-level:

```
@JS('window.navigator.userAgent')
external JSObject get userAgent;
```

Reply:
Could be `JSString` return value?

Reply:
Sure. I should have double-checked the type before posting the example.